stl/inc/__msvc_int128.hpp

```diff
@@ -981,6 +981,14 @@ struct _Unsigned128 : _Base128 {
         _Word[1] |= _That._Word[1];
         return *this;
     }
+
+    _NODISCARD constexpr explicit operator long double() const noexcept {
+        return static_cast<long double>(_Word[1]) * 18446744073709551616.0 + static_cast<long double>(_Word[0]);
+    }
+    template <class _Ty, enable_if_t<is_floating_point_v<_Ty>, int> = 0>
+    _NODISCARD constexpr explicit operator _Ty() const noexcept {
+        return static_cast<_Ty>(static_cast<long double>(*this));
+    }

```

Comment:
It seems better to unify these 2 overloads (ditto for below).
```suggestion
    template <class _Ty, enable_if_t<is_floating_point_v<_Ty>, int> = 0>
    _NODISCARD constexpr explicit operator _Ty() const noexcept {
        return static_cast<_Ty>(_Word[1]) * static_cast<_Ty>(18446744073709551616.0) + static_cast<_Ty>(_Word[0]);
    }
```